### PR TITLE
Added the Cyberlopaedia

### PR DESCRIPTION
**Pull request by cr0mll**

### CODE REVIEW

**Review by LucEast — APPROVED** [review]

**Comment by jknath1** [comment]
**network  ### hack**

**Review by XRjxx — COMMENTED** [review]
Idk what I am doing

**Comment by Mesmorning** [comment]
> **network  ### hack**

**Review by ricky6672 — APPROVED** [review]

**Review by nima0110 — APPROVED** [review]

**Review by ricky6672 — APPROVED** [review]

**Review by ricky6672 — APPROVED** [review]
Good

**Review by akavandal88 — APPROVED** [review]

**Review by dnice1987 — COMMENTED** [review]
```

```

File: README.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ Repository | Description
 [Security Cheatsheets](https://github.com/andrewjkerr/security-cheatsheets) 		| Collection of cheatsheets for various infosec tools and topics
 [Security List](https://github.com/zbetcheckin/Security_list)						 | Great security list for fun and profit
 [Shell](https://github.com/alebcay/awesome-shell) 									| List of awesome command-line frameworks, toolkits, guides and gizmos to make complete use of shell
+[The Cyberclopaedia](https://cr0mll.github.io/cyberclopaedia) | The open-source encyclopaedia of cybersecurity
```
Comment (jp4x4n72) [inline README.md:94]: I believe this section needs to be expanded on.

**Review by dnice1987 — APPROVED** [review]

**Review by nima0110 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nima0110) [reply README.md:94]: ok

**Review by devilish66694 — APPROVED** [review]

**Comment by cr0mll** [comment]
The Cyberclopaedia has been relocated to GitBook.

**Pull request closed without merging**